Client/YooChooseNotifier.php

```diff
@@ -136,7 +134,16 @@ protected function notify(array $events)
 
         $response = $this->guzzle->post(
             $this->getNotificationEndpoint(),
-            array( 'json' => array( 'transaction' => null, 'events' => $events ) )
+            array(
+                'json' => array(
+                    'transaction' => null,
+                    'events' => $events
+                ),
+                'auth' => array(
```

Comment:
Jokes aside, that's odd. Maybe @johaho can confirm the expected auth header format, but I don't see how this call can generate what I think is expected :-)